helm/charts/keto/templates/service-opl.yaml

```
@@ -0,0 +1,22 @@
+{{- if .Values.service.opl.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "keto.fullname" . }}-opl
+  {{- if .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+    app.kubernetes.io/component: opl
+{{ include "keto.labels" . | indent 4 }}
```

```suggestion
    app.kubernetes.io/component: opl
    {{- include "keto.labels" . | nindent 4 }}
```
Please use `nindent` to better manage whitespaces :)